doc/host-setup.md

```diff
@@ -146,3 +146,39 @@ Checking the current value again will reveal that the helper is now disabled
 # cat /sys/module/drbd/parameters/usermode_helper
 disabled
 ```
+
+## Disable Lvmetad in CentOS 7 and Ubuntu 18.04
+
+Reference: https://access.redhat.com/solutions/2053483
```

Comment:
This link requires logging in with a red hat account. I think there is enough information so we can remove it.